network.go

```diff
@@ -2,7 +2,7 @@ package playwright
 
 import "strings"
 
-func serializeHeaders(headers map[string]string) []map[string]string {
+func serializeToNameAndValue(headers map[string]string) []map[string]string {
```

Comment:
```suggestion
func serializeMapToNameAndValue(headers map[string]string) []map[string]string {
```
and put it maybe into helpers.go?

Reply:
Done